traffic_portal/app/src/common/modules/form/deliveryServiceStaticDnsEntry/form.deliveryServiceStaticDnsEntry.tpl.html

```diff
@@ -57,9 +57,18 @@
                 </div>
             </div>
             <div class="form-group" ng-class="{'has-error': hasError(dsStaticDnsEntryForm.address), 'has-feedback': hasError(dsStaticDnsEntryForm.address)}">
-                <label for="address" class="control-label col-md-2 col-sm-2 col-xs-12">Address *</label>
+                <label for="address" class="has-tooltip control-label col-md-2 col-sm-2 col-xs-12">Address *<div class="helptooltip">
+                    <div class="helptext">The Address Rules.
+                        <br>1. Type:A_RECORD, address should be an IPv4 address.
+                        <br>2. Type:AAAA_RECORD, address should be an IPv6 address.
+                        <br>3. Type:CNAME, address must end with a trailing period.
+                        <br>4. Type:TXT_RECORD, address cannot be blank.
+                    </div>
+                </div>
+                </label>
                 <div class="col-md-10 col-sm-10 col-xs-12">
-                    <input id="address" name="address" type="text" class="form-control" ng-model="staticDnsEntry.address" required>
+                    <input id="address" name="address" type="text" class="form-control" ng-model="staticDnsEntry.address"
+                           required title= "Address must be: an IPv4, if type:A_RECORD; an IPv6, if type: AAAA_RECORD; end with a trailing period if type:CNAME_RECORD; cannot be blank; if type: TXT_RECORD">
```

Comment:
i don't think you need this `title` attribute on this input element. do other input elements have it?

Reply:
Yup a lot of them do.. It helps the user no matter where they are hovering, name, textbox
![image](https://user-images.githubusercontent.com/22248619/89679976-5844c600-d8af-11ea-8de9-c5767d7a9f21.png)